### PR TITLE
chore: update ref on functional component

### PR DESCRIPTION
**Pull request by gaoshijun1993**
Add note for ref to functional component

### CODE REVIEW

File: src/v2/api/index.md [hunk 1]
```diff
@@ -2396,7 +2396,11 @@ type: api
 
   An important note about the ref registration timing: because the refs themselves are created as a result of the render function, you cannot access them on the initial render - they don't exist yet! `$refs` is also non-reactive, therefore you should not attempt to use it in templates for data-binding.
 
-- **See also:** [Child Component Refs](../guide/components.html#Child-Component-Refs)
+  > Note: The reference will be HTMLElement when used to functional components because they’re stateless and instanceless.
```
[the remark this answers is not in the document]
Comment (gaoshijun1993) [reply src/v2/api/index.md]: Your opinion is very reasonable, and I've updated it✌

**Comment by zzz6519003** [comment]
❤️

**Comment by sdras** [comment]
Thank you!

**Comment by philBrown** [comment]
Following up from this question on StackOverflow ~ https://stackoverflow.com/questions/61400984/whats-the-reference-refering-to-in-vue-functional-component

What exactly is _"The reference"_ here? I agree with the question asker on SO, it's quite confusing.

**Comment by gaoshijun1993** [comment]
@philBrown Yes you are right, it's quite confusing, I'll open a PR to fix it later

**Pull request merged**